agbenchmark/challenge.py

```diff
@@ -46,6 +46,13 @@ def setup_challenge(self, config: Dict[str, Any]) -> None:
 
         run_agent(self.task, config, self.CHALLENGE_LOCATION)
 
+        # hidden files are added after the agent runs. Hidden files can be python test files.
+        # We copy them in the workspace to make it easy to import the code produced by the agent
+
+        copy_artifacts_into_workspace(
+            config["workspace"], "hidden_files", self.CHALLENGE_LOCATION
```

Comment:
can you write what these hidden_files are in the read me? doesnt have to be structured but for my own understanding what the difference between these and artifacts_out